### PR TITLE
Disable ARM64 jobs

### PR DESCRIPTION
**Pull request by mjroghelia**

### CODE REVIEW

File: .vsts.ci.yml [hunk 1]
```diff
@@ -166,6 +166,7 @@ jobs:
 - job: build_linux_arm64_agent
 ################################################################################
   displayName: Linux Agent (ARM64)
+  condition: false
```
Comment (damccorm) [inline .vsts.ci.yml:169]: Do we want this to be where branch is not equal to master and doesn't start with release?

Comment (damccorm) [reply .vsts.ci.yml:169]: That way we can get ci builds on a different branch for testing

Comment (mjroghelia) [reply .vsts.ci.yml:169]: No, I don't want it running on CI builds for branches for now. We can remove these lines specifically in the branch where we continue testing Arm64.

Comment (damccorm) [reply .vsts.ci.yml:169]: Ok, fair enough

**Pull request merged**